firmware/tunerstudio/rusefi.input

```diff
@@ -1239,10 +1239,10 @@ cmd_stop_engine              = "w\x00\x99\x00\x00"
 	dialog = dwellSettings, "", yAxis
 		panel = dwellCorrection,	Center
 		
-;			Engine->Aux1 Thermistor Settings
+;			Sensors->AUX1 Thermistor Sensor Setting
 	dialog = auxTempSensor1Sensor,	"aux1 Thermistor Settings"
-		field = "!Here is three pairs of thermistor temperature and his resistance."
-		field = "!Typical temperatures is -40°C, 0°C and 100°C"
+		field = "#Here is three pairs of thermistor temperature and his resistance."
+		field = "#Typical temperatures is -40°C, 0°C and 100°C"
```

Comment:
Change color from red ('Warning') to blue('Note')...Again. Some sort of Github glitch?
See https://github.com/rusefi/rusefi/pull/579/commits/d16e5647488a0358cd70238b852da8515238bc2a